src/ApiPort/ApiPort.VisualStudio/ApiPort.VisualStudio.csproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
-    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
```

Comment:
Is it necessary to be on Dev16 in order to open this project? I thought AsyncPackage existed for awhile.  We try to update the solution with released components because this is an oss project that anyone can try to load..

Reply:
I'm testing that for compatibility now - I'll do a revision once I validate everything operates correctly.